systemtest/src/test/java/io/strimzi/systemtest/operators/MultipleClusterOperatorsST.java

```diff
@@ -371,6 +378,7 @@ void deployCOInNamespace(String coName, String coNamespace, List<EnvVar> extraEn
         LOGGER.info("Creating: {} in Namespace: {}", coName, coNamespace);
 
         clusterOperator = clusterOperator.defaultInstallation()
+            .withReconciliationInterval(TestConstants.FASTER_RECONCILIATION_INTERVAL)
```

Comment:
The faster reconciliation interval can cause troubles during the rolling update, as not all of the operations will be done and it will start with another reconciliation.

Does it work for all platforms?

Reply:
Nice catch. I tested it on minikube and OC 4.14 not sure about rest.

Anywya this is not that important to me so i can remove this part with no problem. WDYT ?

Reply:
If it's not needed, you can remove it

Reply:
removed 👍